CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/ViewController.swift

```diff
@@ -0,0 +1,48 @@
+//
```

Comment:
can we add a textView to display the IP address on the device/emulator UI?

Reply:
Done